docs/notebooks/plot_cap.ipynb

```diff
@@ -0,0 +1,796 @@
+{
```

Comment:
<p>Typo: it says g^-1(n) instead of g^-1(eta) </p><p> </p><blockquote>The distribution can also depend on a dispersion parameter&nbsp;...</blockquote><p>While it's correct that Bambi can handle this, the resulting model does not fall under the GLM family -- we call these distributional models.</p><p> </p><p><s>What's more, for these models to be GLMs the conditional distribution of the response must be a member of the exponential family of distributions (however we usually omit this detail). </s> Just saw you also added it, lol.</p><p> </p><blockquote>the link function often bounds the mean outcome between some range or to make it more plausible when used as the parameter in some outcome distribution</blockquote><p>Not sure about the "or to make it more plausible...", maybe you were modifying this and forgot to remove something there?</p><p> </p><p>What do you mean with the following?</p><p> </p><blockquote>there are two ways to interpret the model parameters of a GLM: (1) on the link scale, or (2) on the response scale</blockquote><p> </p><p>I'm not sure to which parameters you refer to (if it's the conditional mean or it's regression coefficients). </p><p> </p><p>The way I see it, you usually have three quantities you can plot with plot_cap</p><p> </p><ul><li>The linear predictor eta.  This always ranges between -inf and inf no matter the nature of the response.</li><li>The mean. Which is mu = g^-1(eta). This is <em>sometimes</em> in the scale of the response variable (e.g. Normal or Poisson likelihood), but other times it's not in the scale of the response (e.g. Binomial likelihood).</li><li>Response values. This are values in the same scale than the response because, well, they are draws from the conditional response distribution for given values of the covariates.</li></ul><p> </p><p>But maybe you tried to say the same thing and I'm just not understanding. Let's open the discussion. </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bambinos/bambi/pull/670/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>